CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.18.4
+  * Facebook busines API to v15.0 [#201](https://github.com/singer-io/tap-facebook/pull/201)
```

Comment:
You bumped it to v14.0, not v15.0

Reply:
@cosimon I went through the link : https://pypi.org/project/facebook-business/
Apparently latest sdk version is 14.0.0 and latest api version is 15.0, their sdk is not up to date with latest api version.
Also, I confirmed that versions previous to v15.0 are deprecated. How shall we proceed with this ?